package/MDAnalysis/core/AtomGroup.py

```diff
@@ -1706,12 +1706,14 @@ def bond(self, pbc=False):
         .. versionadded:: 0.7.3
         .. versionchanged:: 0.8 Added *pbc* keyword
         """
+        from ..core import distances
+
         if len(self) != 2:
             raise ValueError("distance computation only makes sense for a group with exactly 2 atoms")
         if not pbc:
-            return numpy.linalg.norm(self[0].pos - self[1].pos)
+            return util.norm(self[0].pos - self[1].pos)
         else:
-            return MDAnalysis.core.distances.self_distance_array(self.coordinates(), box=self.dimensions)[0]
+            return distances.self_distance_array(self.coordinates(), box=self.dimensions)[0]
```

Comment:
We should have a way to use the parallel distance version, perhaps via a flag in [MDAnalysis.core.flags](https://pythonhosted.org/MDAnalysis/documentation_pages/core/init.html#flags) — something like `False` for always serial, `True` for "use as many threads as you can" and an integer for an upper limit on threads. This should apply not only to your code but also to distance searching in selections (`point` and the non-K-d tree `around`). Raise an issue and reference this issue, too, so that we have an idea where such a change will be needed.